pronto_mia/lib/ui/views/deployment_plan/edit/deployment_plan_edit_view.dart

```diff
@@ -13,12 +12,6 @@ import 'package:pronto_mia/core/models/deployment_plan.dart';
 import 'package:pronto_mia/core/models/simple_file.dart';
 import 'package:pronto_mia/ui/components/form_layout.dart';
 
-@FormView(fields: [

```

Comment:
Why can this be removed?

Reply:
These annotations were used by `stacked_generator` to generate data binding between viewmodels and views for forms.
I've removed `stacked_generator` from our dev dependencies, because it depends on an older version of `stacked`. Since `stacked` is the central library of our application we should keep it as updated as possible.

Since implementing the databinding by hand is easy enough and we don't use the generator for anything else anymore (we used to generate the router and the service registration with it), I think its fair to remove it.